src/main/java/uk/gov/hmcts/reform/sscs/controller/EvidenceUploadController.java

```diff
@@ -0,0 +1,136 @@
+
+@Slf4j
+@RestController
+@RequestMapping("/api/continuous-online-hearings")
+public class EvidenceUploadController {
+    private final EvidenceUploadService evidenceUploadService;
+    private final CoversheetService coversheetService;
+
+    @Autowired
+    public EvidenceUploadController(
+            EvidenceUploadService evidenceUploadService,
+            CoversheetService coversheetService) {
+        this.evidenceUploadService = evidenceUploadService;
+        this.coversheetService = coversheetService;
+    }
+
+    @ApiOperation(value = "Upload evidence",
+            notes = "Uploads evidence for an appeal which will be held in a draft state against the case that is not "
+                    + "visible by a caseworker in CCD. You will need to submit the evidence for it to be visbale in CCD "
+                    + "by a caseworker. You need to have an appeal in CCD."
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Evidence has been added to the appeal"),
+            @ApiResponse(code = 404, message = "No online hearing found with online hearing id"),
+            @ApiResponse(code = 422, message = "The file cannot be added to the document store")
+    })
+    @PutMapping(
+            value = "{identifier}/evidence",
+            consumes = MediaType.MULTIPART_FORM_DATA_VALUE,
+            produces = MediaType.APPLICATION_JSON_VALUE
+    )
+    public ResponseEntity<Evidence> uploadEvidence(
+            @ApiParam(value = "either the online hearing or CCD case id", example = "xxxxx-xxxx-xxxx-xxxx") @PathVariable("identifier") String identifier,
+            @RequestParam("file") MultipartFile file
+    ) {
+        return uploadEvidence(() -> evidenceUploadService.uploadDraftHearingEvidence(identifier, file));
+    }
+
+    private ResponseEntity<Evidence> uploadEvidence(Supplier<Optional<Evidence>> uploadEvidence) {
+        try {
+            Optional<Evidence> evidenceOptional = uploadEvidence.get();
+            return evidenceOptional.map(ResponseEntity::ok)
+                    .orElse(notFound().build());
+        } catch (IllegalFileTypeException exc) {
+            log.info("Cannot upload file illegal file type", exc);
+            return unprocessableEntity().build();
+        }
+    }
+
+    @ApiOperation(value = "List evidence for a hearing",
+            notes = "Lists the evidence that has already been uploaded for a hearing but is still in a draft state."
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "List of draft evidence")
+    })
+    @GetMapping(
+            value = "{identifier}/evidence",
+            produces = MediaType.APPLICATION_JSON_VALUE
+    )
+    public ResponseEntity<List<Evidence>> listDraftEvidence(
+            @PathVariable("identifier") String identifier
+    ) {
+        return ResponseEntity.ok(evidenceUploadService.listDraftHearingEvidence(identifier));
+    }
+
+    @ApiOperation(value = "Submit COR evidence",
```

Comment:
Can we change the API documentation here to remove references to COR and COH and JUI? Assuming this endpoint is still used?